frontend/src/PagesAdmin/SultenMenuAdminPage/SultenMenuAdminPage.tsx

```diff
@@ -57,13 +60,18 @@ export function SultenMenuAdminPage() {
   function menuItemsTableRow(menuItem: MenuItemDto) {
     return [
       dbT(menuItem, 'name'),
-      dbT(menuItem.food_category, 'name'),
+      menuItem.food_category ? dbT(menuItem.food_category as FoodCategoryDto, 'name') : '',
```

Comment:
```suggestion
      menuItem.food_category ? dbT(menuItem.food_category as FoodCategoryDto, 'name') : 'Missing type!',
```
Hvis jeg forstår det rett

Reply:
Er bare error for nå, må fikses i større PR med relation post